regression/iterate_data.yaml

```diff
@@ -0,0 +1,35 @@
+name: streaming-regression-test-iterate-data
+compute:
+  cluster: r1z1
```

Comment:
Please remove the `cluster` name and do something like [this](https://github.com/mosaicml/examples/blob/main/examples/end-to-end-examples/stable_diffusion/yamls/mcloud_run.yaml).